xml/System.Net.WebSockets/ClientWebSocket.xml

```diff
@@ -380,6 +380,20 @@
  For plain WebSocket connections, use the `ws://` scheme in the uri. For secure WebSocket connections over TLS/SSL, use the `wss://` scheme.
 
  This operation will not block. The returned <xref:System.Threading.Tasks.Task> object will complete after the connect request on the <xref:System.Net.WebSockets.ClientWebSocket> instance has completed.
+ 
+ The request has default headers for HTTP/2 request:
+ 
+ :protocol: websocket
+ 
+ For HTTP/1.1 request:
+ 
+ Connection: Upgrade
+ 
+ Upgrade: websocket
+ 
+ Sec-WebSocket-Key: [generated key]
```

Comment:
I believe it's better to start with 1.1 as it's the default one so it's more interesting to the users, and add a bit of clarification on what request we are talking about. I also think it might be better to use `code formatting`.

```suggestion
 The WebSocket handshake request has default headers for HTTP/1.1 request:
 
 `Connection: Upgrade`
 
 `Upgrade: websocket`
 
 `Sec-WebSocket-Key: [generated key]`
 
 For HTTP/2 request:
 
 `:protocol: websocket`
```